org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServiceAccessor.java

```diff
@@ -174,8 +171,7 @@ public static void enableLanguageServerContentType(
 					LanguageServerDefinition lsDefinition = contentTypeToLSDefinition.getValue();
 					IContentDescription contentDesc = editorFile.getContentDescription();
 					if (contentTypeToLSDefinition.isEnabled() && contentType != null && contentDesc != null
-							&& contentType.equals(contentDesc.getContentType())
-							&& lsDefinition != null) {
+							&& contentType.equals(contentDesc.getContentType()) && lsDefinition != null) {
```

Comment:
the code was already like this, but `lsDefinition != null` can be already checked before calling `editorFile.getContentDescription();`